source/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
```

Comment:
You can change the package name to source, It is neater when calling it `source.Config`. Also this will not make conflict with other config imports.